setup.py

```diff
@@ -70,7 +70,6 @@
                       'corner>=2.0.1',
                       #'scikit-learn>=0.17.0',  # travis does not like scikit-learn
                       ]
```

Comment:
But that directive has been deprecated in both setuptools and pip. We shouldn't have it there if it's a broken noop. If you want it in setup.py, the right thing to do is release a ``pycbc-mpld3`` in PyPi and then add it to the regular requirements.